editors/code/src/installation/language_server.ts

```diff
@@ -105,6 +105,8 @@ export async function ensureLanguageServerBinary(
                     `GitHub repository: ${err.message}`
                 );
 
+                console.error(err);
```

Comment:
I'm not 100% sure we need to write the error out since the message is shown in vscode but I don't feel strongly about it. If you think it makes sense then that's fine by me.

Reply:
@kjeremy `err.message` is as less verbose as it can, `err.stack` and `console.error(err)` are for the better debugging experience.
Here is the comparison:
![image](https://user-images.githubusercontent.com/36276403/74282662-32d46680-4d29-11ea-8737-5725371f4164.png)